example_gh_test.go

```diff
@@ -153,6 +154,43 @@ func ExampleGQLClient_advanced() {
 	fmt.Println(query)
 }
 
+// Add a star to the cli/go-gh repository using the GQL API.
+func ExampleGQLClient_Mutate_simple() {
+	client, err := gh.GQLClient(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var m struct {
+		AddStar struct {
+			Starrable struct {
+				Repository struct {
+					Stargazers struct {
+						Nodes []struct {
+							Login string
```

Comment:
Why query the information about login names for the latest 10 stargazers? I don't feel that helps the example, and instead just adds noise. Maybe we could just fetch StargazerCount like for gists and display that as the result of the mutation

Reply:
I have come across this somewhat regularly when using the apis, for example get the issues in a ProjectsV2 or the comments in a PullRequest, and wanted to showcase how that is used in a mutation. I do see that there are existing examples of it in `ExampleGQLClient_simple` and `ExampleGQLClient_advanced`, I'll remove it here

Reply:
oh - I also wanted to show the use of `first`, separate from `input`, which I don't see in other examples and is often required by the api. I removed it in a77f72b if you prefer without it, or we can revert it if you think it adds something